src/components/navigation/NavLink.tsx

```diff
@@ -0,0 +1,45 @@
+import { SupportedLanguageTypes } from '@/configs'
+import { hulen_yellow, hulen_yellow_text } from '@/styles'
+import { SanityNavElement } from '@/types/sanity'
+import { ArrowDownward } from '@mui/icons-material'
+import { Typography } from '@mui/material'
+import Link from 'next/link'
+import { MouseEventHandler } from 'react'
+
+export const NavLink = ({
+  navElement,
+  language,
+  isCurrentPath,
+  onClick,
+}: {
+  navElement: SanityNavElement
+  language: SupportedLanguageTypes
+  isCurrentPath: boolean
+  onClick: MouseEventHandler
+}) => {
+  return (
+    <Link
```

Comment:
wrapper teknikken for link, kjør litt for mye typografi styling her som link ikke burda ha ansvar for som kanskje vil automagisk løse seg ved å gå over til wrapper metoden, hvis ikke er det typography som burde ha ansvar for å definere typografi, og det burde skje i theme, ikke lokalt